### ISSUE TITLE
DRAM-setup.py broke when upgrading to v1.5.0

### ISSUE THREAD

**Issue by cjburgess** [WrightonLabCSU/DRAM#326]
I can't seem to get DRAM-setup.py to work after upgrading to the newest DRAM version.

```bash
❯ DRAM-setup.py -h
  File "/home/christopher.burgess/.conda/envs/DRAM/bin/DRAM-setup.py", line 124
    set_db_locs_parser.add_argument('--camper_tar_gz_loc', default=None,
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
```
I've tried completely uninstalling the environment and cleaning my conda cache with no luck. Is this a problem on my end or was there a bug in the release?

**Comment by cjburgess** [comment]
Quick update, I took a peak at `DRAM-setup.py` and there is definately a syntax error there on line 122 (below). It looks like the camper argument help was pasted in the wrong spot.

```python
    set_db_locs_parser.add_argument('--viral_loc', default=None,
    set_db_locs_parser.add_argument('--camper_tar_gz_loc', default=None,
                                    help='The source for the CAMPER database files, this is a tar.gz file downloaded'
                                    ' from the release page https://github.com/WrightonLabCSU/CAMPER/releases')
                                    help='mmseqs2 database file from ref seq viral gene collection')
```

Here is the fixed code:

```python
    set_db_locs_parser.add_argument('--viral_loc', default=None,
                                    help='mmseqs2 database file from ref seq viral gene collection')
    set_db_locs_parser.add_argument('--camper_tar_gz_loc', default=None,
                                    help='The source for the CAMPER database files, this is a tar.gz file downloaded'
                                    ' from the release page https://github.com/WrightonLabCSU/CAMPER/releases')
```

**Comment by smdabdoub** [comment]
This was fixed in commit 9ff00c9
